db/inmemory_test.go

```diff
@@ -0,0 +1,54 @@
+package db
+
+import "testing"
+
+func TestBasic(t *testing.T) {
+	im := NewInMemoryVolumeDatabase()
+
+	err := im.Connect()
+	if err != nil {
+		t.Fatal("Connect returned an error! " + err.Error())
+	}
+
+	list, err := im.List()
+	if err != nil {
+		t.Fatal("List returned an error! " + err.Error())
```

Comment:
I think I see what you mean now. It's harder for me to see what these tests are designed for since there aren't any comments.

This specific `func` is fine then